LambdaEngine/Include/Rendering/PaintMaskRenderer.h

```diff
@@ -115,7 +123,7 @@ namespace LambdaEngine
 		TArray<TSharedRef<Buffer>>	m_UnwrapDataCopyBuffers;
 		TSharedRef<Buffer>			m_UnwrapDataBuffer = nullptr;
 
-		const DrawArg*												m_pDrawArgs;
+		const DrawArg*												m_pDrawArgs = nullptr;
```

Comment:
Is not really unsafe to store a DrawArg? Since there is a huge risk of these being recreated every frame?

Reply:
The DrawArg is being set every time the UpdateDrawArgs function is called, which to my knowledge means that it shouln't happen. @jonathan2222 probably knows more of the subject if you want to address it.

Reply:
Oki, just curious 😊